package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "@nexusmutual/sdk",
-  "version": "0.2.30",
+  "version": "0.3.0",
   "description": "Nexus Mutual SDK",
   "scripts": {
     "build": "node scripts/build.js",
+    "lint": "eslint --max-warnings 0 .",
+    "lint-fix": "npm run lint -- --fix",
```

Comment:
Minor, but could we make the command `lint:fix` instead? We use this notation in all frontend projects

Reply:
done